build.sbt

```diff
@@ -99,6 +101,26 @@ lazy val enumeratum = project
   )
   .dependsOn(core)
 
+lazy val http4s = project
+  .in(file("modules/http4s"))
+  .settings(
+    moduleName := "ciris-http4s",
+    name := moduleName.value,
+    dependencySettings ++ Seq(
+      libraryDependencies ++= Seq(
+        "org.http4s" %% "http4s-core" % http4sVersion,
+        "org.tpolecat" %% "typename" % typeNameVersion
```

Comment:
`typename` used to omit hardcoding Uri type (`org.http4s.Uri`) manually.
I like the option I did, but also not against hardcoding `org.http4s.Uri` if we want to omit additional dependency.

Reply:
I removed the `typename` dependency as it's fine to just use `Uri` as name.